src/sentry/notifications/notificationcontroller.py

```diff
@@ -53,19 +56,29 @@ class NotificationController:
 
     def __init__(
         self,
-        recipients: Iterable[RpcActor] | Iterable[Team] | Iterable[RpcUser],
+        recipients: list[Recipient],
         project_ids: Iterable[int] | None = None,
         organization_id: int | None = None,
         type: NotificationSettingEnum | None = None,
         provider: ExternalProviderEnum | None = None,
     ) -> None:
-        self.recipients = recipients
+        org = Organization.objects.filter(id=organization_id).first()
```

Comment:
need to remove this since can't use this in this silo, but how can i get the org for the feature flag?